lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -953,6 +958,28 @@ ChangeResult TypeAnalyzer::visitAtenMaxPool2dOp(
   return getLatticeElement(op->getResult(0)).join(knowledge);
 }
 
+ChangeResult TypeAnalyzer::visitAtenConstantPadNdOp(
+    AtenConstantPadNdOp op,
+    ArrayRef<LatticeElement<ValueKnowledge> *> operands) {
+  auto knowledge =
+      ValueKnowledge::getNotNonePessimisticValueState(op->getContext());
+  knowledge.hasSizes = true;
+  auto &ifm = operands[0]->getValue();
```

Comment:
we don't need the & as the return type of `getValue` is already reference type.

Reply:
nit: not sure why is the variable name `ifm`? Can we call it `input`?

Reply:
> we don't need the & as the return type of `getValue` is already reference type

While that holds true for `auto` and pointer types, the `&` is necessary if you want a reference rather than a copy (e.g. 
https://github.com/llvm/torch-mlir/pull/515#discussion_r781478778).

Reply:
ha, you are right.